src/drm/widevineConfig.ts

```diff
@@ -31,19 +33,25 @@ export interface WidevineConfig {
    * Note that both the passed `license` data and this block return value should be a Base64 string.
    * So use whatever solution suits you best to handle Base64 in React Native.
    *
+   * @platform Android
+   *
    * @param license - Base64 encoded license data.
    * @returns The processed Base64 encoded license.
    */
   prepareLicense?: (license: string) => string;
   /**
    * Set widevine's preferred security level.
+   *
+   * @platform Android
    */
   preferredSecurityLevel?: string;
   /**
    * Indicates if the DRM sessions should be kept alive after a source is unloaded.
    * This allows DRM sessions to be reused over several different source items with the same DRM configuration as well
    * as the same DRM scheme information.
    * Default: `false`
+   *
+   * @platform Android
    */
-  shouldKeepDrmSessionsAlive: boolean;
+  shouldKeepDrmSessionsAlive?: boolean;
```

Comment:
NIT: This is not ideal because it suggests that this is not mandatory on Android. But I don't think there is a better solution without duplicating a lot of code.

Reply:
@strangesource It is not "mandatory to set" as it already has a default value in the Android Player SDK.
Therefore to enable using default value, without knowing it, we should make this optional to avoid duplicating default values.